### PR TITLE
Add labels and add translations

### LINKED ISSUE #605: Add/translate labels for all forms
A few of these lack labels because I removed the behavior that labels default to the placeholder, others just need translation:

TODO:
- [x] EventEditor (translations)
- [x] AdminRegisterForm (no labels)
- [x] UserSettings (translations)
- [x] UserConfirmation (no labels)
- [x] StudentConfirmation (missing labels)
- [x] GalleryPictureEditModal (missing labels)
- [x] JoblistingEditor (missing labels)
- [x] MeetingEditor (missing labels)
- [x] AddQuote (missing labels)

I didn't go through these too closely, so feel free to check them off if they should be ignored.

### PR DESCRIPTION
**Pull request by SmithPeder**
Fixes #605
This is not a complete fix for #605, but a most of it, the things that i where unable to fix are now in #871

### CODE REVIEW

File: app/routes/events/components/EventAdministrate/AdminRegisterForm.js [hunk 1]
```diff
@@ -23,18 +23,21 @@ const AdminRegister = ({
       <form onSubmit={handleSubmit}>
         <Field
           placeholder="Begrunnelse"
+          lable="Begrunnelse"
```
Comment (odinuge) [inline app/routes/events/components/EventAdministrate/AdminRegisterForm.js]: Skriveleif, `lable`?

File: app/routes/events/components/EventAdministrate/AdminRegisterForm.js [hunk 2]
```diff
@@ -23,18 +23,21 @@ const AdminRegister = ({
...
           name="reason"
           component={TextEditor.Field}
         />
         <Field
           placeholder="Tilbakemelding"
+          lable="Tilbakemelding"
```
Comment (odinuge) [inline app/routes/events/components/EventAdministrate/AdminRegisterForm.js]: Skriveleif, `lable`?

File: app/routes/joblistings/components/JoblistingEditor.js [hunk 3]
```diff
@@ -113,7 +113,7 @@ function JoblistingEditor({
           component={DatePicker.Field}
         />
         <Field
-          placeholder="Arbeidssteder"
+          placeholder="Steder stilling er tilgjengelig"
```
Comment (edvardvb) [inline app/routes/joblistings/components/JoblistingEditor.js]: I think this one is a bit clunky, is it really a better description?

File: app/routes/joblistings/components/JoblistingEditor.js [hunk 4]
```diff
@@ -137,13 +137,13 @@ function JoblistingEditor({
...
-          placeholder="Søknadslenke"
+          placeholder="Lenke til søknad"
           label="Søknadslenke"
           name="applicationUrl"
           component={TextInput.Field}
         />
         <Field
-          placeholder="Kontaktperson"
+          placeholder="Person som kan kontaktes angående annonsen"
           label="Kontaktperson"
```
Comment (edvardvb) [inline app/routes/joblistings/components/JoblistingEditor.js]: Same

File: app/routes/joblistings/components/JoblistingEditor.js [hunk 5]
```diff
@@ -137,13 +137,13 @@ function JoblistingEditor({
...
         <Field
-          placeholder="Søknadslenke"
+          placeholder="Lenke til søknad"
           label="Søknadslenke"
           name="applicationUrl"
           component={TextInput.Field}
         />
         <Field
-          placeholder="Kontaktperson"
+          placeholder="Person som kan kontaktes angående annonsen"
```
Comment (edvardvb) [inline app/routes/joblistings/components/JoblistingEditor.js]: Same

File: app/routes/joblistings/components/JoblistingEditor.js [hunk 6]
```diff
@@ -153,14 +153,14 @@ function JoblistingEditor({
...
-          placeholder="Søknadsintro"
+          placeholder="Intro til søknaden"
           component={EditorField.Field}
         />
         <Field
           name="text"
           className={styles.textField}
           label="Søknadstekst:"
-          placeholder="Søknadstekst"
+          placeholder="Hoveddel av søknaden"
```
Comment (edvardvb) [inline app/routes/joblistings/components/JoblistingEditor.js]: Same

File: app/routes/events/components/EventAdministrate/AdminRegisterForm.js [hunk 7]
```diff
@@ -23,18 +23,21 @@ const AdminRegister = ({
...
           placeholder="Tilbakemelding"
+          label="Tilbakemelding"
           name="feedback"
           component={TextEditor.Field}
         />
         <Field
           name="pool"
           component={SelectInput.Field}
           placeholder="Pool"
+          label="Pool"
```
Comment (ekmartin) [inline app/routes/events/components/EventAdministrate/AdminRegisterForm.js]: I think we should call pool something more understandable to the user - "Påmeldingsgruppe" or something like that? cc @edvardvb @hanswilw

Comment (edvardvb) [reply app/routes/events/components/EventAdministrate/AdminRegisterForm.js]: Hmm, I prefer 'Pulje', but it might not be very understandable either

Comment (ekmartin) [reply app/routes/events/components/EventAdministrate/AdminRegisterForm.js]: I'll merge, then we can fix later.

**Pull request merged**